cv/CV-guide.md

```diff
@@ -0,0 +1,171 @@
+```
+[Term]
+id: QC:4000111
+name: Number of MS1 spectra
+def: "Number of MS1 spectra" [PSI:QC]
+comment: A lower number of MS1 spectra acquired during one sample run compared to similar runs can indicate mismatched instrument settings or issues with the instrumentation.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+has_units: STATO:0000047! count
+```
+to define your metric object which would look like 
+```
+{
+    "accession": "QC:4000111",
+    "name": "Number of MS1 spectra",
+    "value": 7130,
+}
+```
+where the value would then be implicitly interpreted as of type `count`.
+
+### n-tuple:
+If we choose to write a n-tuple type, for example:
+```
+[Term]
+id: QC:4000113
+name: MZ acquisition range
+def: "Upper and lower limit of m/z values at which spectra are recorded." [PSI:QC]
+comment: Acquisition levels can be used as a criterion to assess the comparability of instrument settings between runs. 
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units MS:1000040 ! m/z
+```
+Then we can assign the tuples as an n-array and their units implicitly assumed to be of `m/z` unit like this:
+```
+{
+    "accession": "QC:4000113",
+    "name": "MZ acquisition range",
+    "value": [400,1200],
+}
+```
+
+
+### matrix:
+TBD
+
+### table:
+If we want to report multiple types of values for multiple observations, we will choose a table. A table will be defined by it's columns, which can be optional. Here is an example
+```
+[Term]
+id: QC:4000095
+name: Sampling rates
+definition: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
+comment: The sampling rate may give insights to peptide separation, dynamic exclusion settings and gradient efficiency.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column_type: QC:4000089 ! Sample rates
+has_column_type: QC:4000090 ! Counts
+has_optional_column_type: QC:4000091 ! Peptide sequences
+```
+
+Any metric object of the type `Sampling rates` will be expected to contain a column of name `Sample rates`  and `Counts` and may have `Peptide sequences`. These define the respective column values as follows.
+
+```
+[Term]
+id: QC:4000089
+name: Sample rates
+def: "The column contains sample rates as number of occurrences." [PSI:QC]
+is_a: QC:4000082 ! Defined value type
+has_units: PATO:0000161 ! rate
+
+[Term]
+id: QC:4000090
+name: Counts
+def: "The column contains counts of occurrences of a common type." [PSI:QC]
+is_a: QC:4000082 ! Defined value type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000091
+name: Peptide sequences
+def: "The column contains peptide sequences." [PSI:QC]
```

Comment:
`. Multiple peptides can be separated by space`

Reply:
As soon as the term is fixed in the obo, this'll get an update